src/test/kotlin/no/nav/syfo/testutil/InternalMockEnvironment.kt

```diff
@@ -43,6 +44,10 @@ class InternalMockEnvironment private constructor() {
         clientEnvironment = environment.clients.arbeidsuforhetvurdering,
         httpClient = externalMockEnvironment.mockHttpClient
     )
+    private val manglendeMedvirkningClient = ManglendeMedvirkningClient(
+        azureAdClient = azureAdClient,
+        clientEnvironment = environment.clients.manglendeMedvirkning,
```

Comment:
Burde man sende inn `httpClient = externalMockEnvironment.mockHttpClient` her også? Har tydeligvis ikke noe å si for testene da...